src/backend/modules/s3/s3-module.ts

```diff
@@ -19,19 +19,24 @@ export const client = new S3Client({
 });
 
 export const loadPic = async (key: string, data: PutObjectRequest['Body'] | string, contentType: string) => {
-    await client.send(
-        new PutObjectCommand({
-            Bucket: config.s3.bucket,
-            BucketKeyEnabled: false,
-            Key: `section/${key}`,
-            Body: data,
-            ACL: 'authenticated-read',
-            ContentType: contentType,
-            CacheControl: 'no-cache',
-        }),
-    );
-
-    return `${config.s3.endpoint}/${config.s3.bucket}/${key}`;
+    try {
+        await client.send(
+            new PutObjectCommand({
+                Bucket: config.s3.bucket,
+                BucketKeyEnabled: false,
+                Key: `section/${key}`,
+                Body: data,
+                ACL: 'authenticated-read',
+                ContentType: contentType,
+                CacheControl: 'no-cache',
+            }),
+        );
+
+        return;
+    } catch (error) {
+        // @ts-ignore
+        return error.message as string;
```

Comment:
How to distinguish success from error? They both are string

Reply:
success will return undefined, after I'll correct catch to `error instanceof Error ? error.message : 'Upload error'` fail will be always string